.github/workflows/build-test-distribute.yaml

```diff
@@ -111,6 +111,8 @@ jobs:
           key: ${{ runner.os }}-devtools-${{ hashFiles('mk/dependencies/deps.lock') }}
           restore-keys: |
             ${{ runner.os }}-devtools
+      - run: |
+          make dev/tools
```

Comment:
Did it ever work successfully?

Reply:
It does if you are lucky enough to already have the cache :)